### PR TITLE
Add support for TOS and handle device removal

### PR DESCRIPTION
**Pull request by hselasky**

### CODE REVIEW

File: krping.c [hunk 1]
```diff
@@ -1924,6 +1926,10 @@ static void krping_run_client(struct krping_cb *cb)
 	struct ib_recv_wr *bad_wr;
 	int ret;
 
+	/* set type of service, if any */
+	if (cb->tos != 0)
```
Comment (larrystevenwise) [inline krping.c:1932]: Is 0 not a valid service type?

**Review by larrystevenwise — COMMENTED** [review]
The device removal commit looks fine.

File: krping.c (hunk 1 above)
Comment (hselasky) [reply krping.c:1932]: 0 is the default TOS, no need to set it from what I can see.

Comment (larrystevenwise) [reply krping.c:1932]: ok then.  The PR looks good. Thanks!

**Comment by larrystevenwise** [comment]
Merged...thanks.

**Pull request merged**